source/about.html.erb

```diff
@@ -62,16 +60,16 @@ title: "A framework for creating ambitious web applications."
 <div class="features section">
   <h2>More Productive Out of the Box.</h2>
   <div class="feature">
-    <div class="handlebars"></div>
-    <p>Write dramatically less code with Ember's Handlebars integrated templates that update automatically when the underlying data changes.
+    <div class="productivity"></div>
+    <p>Ember.js is built for productivity. Designed with developer ergonomics in mind, its friendly APIs help you get your job done&mdash;fast.</p>
```

Comment:
I was thinking about the first two images (and text) as Ember's key values: 
* **Productivity**- developer ergonomics, friendly APIs
* **Best practices built in** - Stopping the wheel invention

And then the Handlebars image/text is there, which doesn't really embrace a library value. Don't get me wrong, I love handlebars (and what HtmlBars will offer too), but it didn't "pop" to me as a core value, so I moved it and liked how it "felt" better :)